autode/species/species.py

```diff
@@ -496,6 +496,29 @@ def enthalpy(self) -> Optional[val.Enthalpy]:
             logger.warning('Could not calculate H - an energy was None')
             return None
 
+    @property
+    def zpe(self) -> Optional[val.Energy]:
+        """
+        Zero point vibrational energy of this species
+
+        -----------------------------------------------------------------------
+        Returns:
+            (autode.values.Energy | None): ZPE if frequencies are defined
+        """
+        if self.n_atoms < 2:
```

Comment:
imo its better here to say `if self.atoms == 1`, as I think thats what you mean?

Reply:
There also could be `self.n_atoms == 0` so I think this is right, despite the comment.. (will update it to be more informative) 👌